### PR TITLE
Move comments to a new line for PowerShell concepts

### PR DESCRIPTION
**Pull request by Prince25**
### Description
Since I was credited as "Anonymous" for contributing to PowerShell docs, @yangc95 [asked me](https://github.com/Codecademy/docs/pull/2185#issuecomment-1556279892) to move comments to a new line and push the changes to the main branch.

### Type of Change
- Updating the documentation

### Checklist
- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/powershell/concepts/operators/operators.md [hunk 1]
```diff
@@ -94,8 +103,13 @@ Assignment operators can be used to assign, change, or append values to variable
 
 ```shell
 PS > $number = 4
-PS > $number += 6 # $number is now 10
```
Comment (SSwiniarski) [inline content/powershell/concepts/operators/operators.md:97]: Again, we can't really move the comments in the command line examples.

File: content/powershell/concepts/operators/operators.md [hunk 2]
```diff
@@ -150,14 +164,21 @@ Precedence order is the order in which PowerShell evaluates the operators if mul
 ```shell
 PS > $num_1 = 4
 PS > $num_2 = 3
-PS > $num_1 -lt $num_2  # num_1 is not less than num_2
```
Comment (SSwiniarski) [inline content/powershell/concepts/operators/operators.md:153]: Again I'd undo the command line examples

File: content/powershell/concepts/operators/operators.md [hunk 3]
```diff
@@ -150,14 +164,21 @@ Precedence order is the order in which PowerShell evaluates the operators if mul
...
+PS > $num_1 -lt $num_2
 False
+# num_1 is not less than num_2
 
-PS > $num_2++           # num_2 is now 4
```
Comment (SSwiniarski) [inline content/powershell/concepts/operators/operators.md:156]: See prior comments.

File: content/powershell/concepts/operators/operators.md [hunk 4]
```diff
@@ -150,14 +164,21 @@ Precedence order is the order in which PowerShell evaluates the operators if mul
...
+# num_2 is now 4
+
+PS > $num_1 -ge $num_2
 True
+# num_1 is greater than or equal to num_2
+
+PS > $num_1 /= 2
+# num_1 is now 2
 
-PS > $num_1 /= 2        # num_1 is now 2
```
Comment (SSwiniarski) [inline content/powershell/concepts/operators/operators.md:160]: See prior comments.

File: content/powershell/concepts/variables/variables.md [hunk 5]
```diff
@@ -54,7 +54,8 @@ PS > [Int]$age = 25
 PS > $age
 25
 
-PS > [Int]$age = "twenty five" # Results in an error
```
Comment (SSwiniarski) [inline content/powershell/concepts/variables/variables.md:57]: Let's not move the comment for the command line examples.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Prince25 I did a first review and left some comments. They're all about moving the comments from the command line examples. You can move the comments in the scripting samples, but in the command line it breaks the example. The good news is that all the files where this happens, there seems to be a scripting example, so fixing this wont break what you're trying to do.

**Comment by Prince25** [comment]
Thank you. I believe I reverted all of the command line examples but feel free to take another look.

**Review by SSwiniarski — APPROVED** [review]
@Prince25 Looks good!

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@Prince25 edits will appear here: powershell: [Arrays](https://www.codecademy.com/resources/docs/powershell/arrays), [Conditionals](https://www.codecademy.com/resources/docs/powershell/conditionals), [Objects](https://www.codecademy.com/resources/docs/powershell/objects), [Variables](https://www.codecademy.com/resources/docs/powershell/variables)

cc: @SSwiniarski

**Pull request merged**